algorithms/MDLMDec/loader_.py

```
@@ -0,0 +1,338 @@
+        
+        #########################
+        # Map to format outputs #
+        #########################
+        lambda_function = lambda example: map_fn(
+            example,
+            tokenizer=self.tokenizer,
+            dic=self.amod_dic,
+            top=top_pks, 
+            max_seq=max_seq,
+            reverse=reverse,
+        )
```

Ruff (0.16.1)

[error] Do not assign a `lambda` expression, use a `def`

Rewrite `lambda_function` as a `def`

(E731)